src/rules/style.ts

```diff
@@ -43,6 +43,8 @@ export default {
   'no-consecutive-blank-lines': [true, 2],
   // Disallows parameter properties in class constructors.
   'no-parameter-properties': false,
+  // Don’t if you import “foo” anyway.
```

Comment:
Yup, that what it seems to be doing. 
https://github.com/palantir/tslint/blob/master/test/rules/no-reference-import/test.d.ts.lint